test/functional/wallet_listtransactions.py

```diff
@@ -275,6 +277,13 @@ def normalize_list(txs):
         assert_equal(['pizza2'], self.nodes[0].getaddressinfo(addr2)['labels'])
         assert_equal(['pizza3'], self.nodes[0].getaddressinfo(addr3)['labels'])
 
+    def run_invalid_parameters_test(self):
+        self.log.info("Test listtransactions RPC parameter validity")
+        assert_raises_rpc_error(-8, 'Label argument must be a valid label name or "*"', self.nodes[0].listtransactions, label="")
+        self.nodes[0].listtransactions(label="*")
```

Comment:
I didn't understand this line, I think you're not using the return of this `listtransactions`. Noticed that removing it, the test doesn't fail.

Reply:
The intention here is to ensure that passing "*" as label is considered valid; the asterisk is not passed in any other part of the tests, so I thought it would make sense to do it here. I agree though that it looks a bit odd.